recipes/glibmm/all/conanfile.py

```diff
@@ -75,7 +75,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("glib/2.77.2")
+        self.requires("glib/2.78.0")
```

Comment:
```suggestion
        self.requires("glib/2.78.0", transitive_headers=True)
```

Reply:
`transitive_libs=True` may also be required, let's see how it goes first with `transitive_headers=True` only.